nixos/doc/manual/configuration/x-windows.xml

```diff
@@ -159,21 +158,14 @@
  <simplesect xml:id="sec-x11--graphics-cards-amd">
   <title>Proprietary AMD drivers</title>
   <para>
-   AMD provides a proprietary driver for its graphics cards that has better 3D
-   performance than the X.org drivers. It is not enabled by default because
-   it’s not free software. You can enable it as follows:
+   AMD provides a proprietary driver for its graphics cards that is not
+   enabled by default because it’s not free software.
```

Comment:
AMD provides a proprietary driver for its graphics cards that is not enabled by default because it’s not free software and also it is often broken and doesn't necessary offer more features or performance. If you want to use it anyway you need to explicitly set it as: